### ISSUE TITLE
tomb index / search and mlocate

### ISSUE THREAD

**Issue by nerun** [dyne/Tomb#464]
Please, remove the need to just `mlocate` to be installed, instead add some alternatives like `locate`, `plocate` etc.
```
$ tomb index
tomb [E] Index command needs 'mlocate' to be installed.
$ tomb search "test"
tomb [E] Index command needs 'mlocate' to be installed.
```

In Linux Mint, `mlocate` is just a dummy package. I have installed but when use mlocate in terminal, nothing happens:
```
$ mlocate
zsh: command not found: mlocate
```
Even if i add a script pointing to locate, it doesn't works:
```shell
#!/usr/bin/env zsh
locate "$@"
```
Would be better if `tomb` consider some alternatives.

**Comment by nerun** [comment]
```
$ tomb --version
  Tomb 2.9.0 - a strong and gentle undertaker for your secrets

   Copyright (C) 2007-2021 Dyne.org Foundation, License GNU GPL v3+
   This is free software: you are free to change and redistribute it
   For the latest sourcecode go to <http://dyne.org/software/tomb>

   This source code is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
   When in need please refer to <http://dyne.org/support>.

  System utils:

  zsh 5.8.1 (x86_64-ubuntu-linux-gnu)
  Sudo versão 1.9.9
  cryptsetup 2.4.3
  pinentry-gnome3 (pinentry) 1.1.1
  findmnt de util-linux 2.37.2
  gpg (GnuPG) 2.2.27 - key forging algorithms (GnuPG symmetric ciphers):
  IDEA 3DES CAST5 BLOWFISH AES AES192 AES256 TWOFISH CAMELLIA128 CAMELLIA192 CAMELLIA256

  Optional utils:

  /usr/bin/gettext
  dcfldd not found
  /usr/bin/shred
  /usr/bin/steghide
  /usr/sbin/resize2fs
  /usr/libexec/tomb/tomb-kdb-pbkdf2
  /usr/bin/qrencode
  swish-e not found
  unoconv not found
  /usr/bin/lsof
```

**Comment by nerun** [comment]
Fixed in PR #483
